pkg/components/rook/component.go

```diff
@@ -31,6 +32,7 @@ func init() {
 type component struct {
 	Namespace                string            `hcl:"namespace,optional"`
 	NodeSelectors            map[string]string `hcl:"node_selector,optional"`
+	NodeSelectorString       string
```

Comment:
If the "structured" field is called `NodeSelectors`, why this is not `NodeSelectorsString`? Should we rename either one or anotoher to match?

Also, in other components, we use `Raw` suffix for converted fields, I suggest using the same here.